.github/scripts/gql_mocks.json

```diff
@@ -25831,6 +25831,9 @@
           "changedFiles": 3,
           "files": {
             "nodes": [
+              {
+                "path": "aten/src/ATen/native/Convolution.cpp"
+              },
```

Comment:
was this edited directly? if yes, theres a chance that it might disappear because we sometimes advise deleting the entire gql_mocks file and regenerating the entire thing

Reply:
Good point, this file needs to be regenerated (or new test case should be added)

Reply:
Good catch! It's indeed edited directly (to serve as a test case).  Let me try to add that PR https://github.com/pytorch/pytorch/pull/95233 with duplicated files then, so regenerating the file would work too.